### PR TITLE
feat(react-components): added reset camera position to scene data position

### PR DESCRIPTION
**Pull request by pramodcog**
#### Type of change

Feat  

#### Jira ticket :blue_book:

https://cognitedata.atlassian.net/browse/BND3D-2293

## Description :pencil:
This component button reset the camera position to the default/home position of the scene if available else it would fit to all models

## How has this been tested? :mag:

In storybook

## Test instructions :information_source:

- `yarn && yarn build && yarn storybook`
- Load `SceneContainer` example and select `reset camera button`

## Checklist :ballot_box_with_check:

- [x] I am proud of this feature.
- [x] I have performed a self-review of my own code.
- [x] I have added PR type (Feat, Bug, Chore, Test, Docs, Style, Refactor) to the PR title.
- [ ] I have manually tested this for different scenarios (different models, formats, devices, browsers).
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the public documentation.
- [ ] I have added unit and visuals tests to prove that my feature works to the best of my ability.
- [ ] I have refactored the code for readability to the best of my ability.
- [ ] I have checked that my changes do not introduce regressions in the public documentation.
- [ ] I have outlined any known defects / lacking capabilities in the contents of this PR.
- [ ] I have listed any remaining work that should follow this PR in the description or jira/miro/etc.
- [ ] I have added TSDoc to any public facing changes.
- [ ] I have added "developer documentation" in any package README.md that is related / applicable to this PR.
- [ ] I have noted down and am currently tracking any technical debt introduced in this PR.
- [ ] I have thoroughly thought about the architecture of this implementation.
- [ ] I have asked peers to test this feature.

### CODE REVIEW

File: react-components/src/components/RevealToolbar/ResetCameraButton.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+export const ResetCameraButton = ({
+  sceneExternalId,
+  sceneSpaceId
+}: ResetCameraButtonProps): ReactElement => {
+  const { t } = useTranslation();
+  const cameraNavigation = useCameraNavigation();
+
+  const resetCameraToHomePosition = useCallback(() => {
+    if (sceneExternalId !== undefined && sceneSpaceId !== undefined) {
+      useSceneDefaultCamera(sceneExternalId, sceneSpaceId);
```
Comment (Savokr) [inline react-components/src/components/RevealToolbar/ResetCameraButton.tsx:26]: I think its forbidden by react to do something like this. You can't run hooks conditionally. Also the API for the hook is the same as for main camera navigation hook, i.e. you should destructure or directly call the function that hook returns, not just call the hook itself. Also, scene that you can test it with on 3D test is called `savelii_scene1`, it has some default camera position.

**Review by Savokr — DISMISSED** [review]
Comment related to the scene camera hook.

File: react-components/src/components/RevealToolbar/ResetCameraButton.tsx [hunk 2]
```diff
@@ -20,13 +19,15 @@ export const ResetCameraButton = ({
 }: ResetCameraButtonProps): ReactElement => {
   const { t } = useTranslation();
   const cameraNavigation = useCameraNavigation();
+  const resetToDefault = useSceneDefaultCamera(sceneExternalId ?? '', sceneSpaceId ?? '');
```
Comment (haakonflatval-cognite) [inline react-components/src/components/RevealToolbar/ResetCameraButton.tsx:22]: Can we rename this variable? Or instead write `const { fitCameraToSceneDefault } = useSceneDe....`?

File: react-components/tests/visual-tests/__screenshots__/Toolbar.stories.spec.ts/cad-model-container-storybook-1.png
Comment (haakonflatval-cognite) [inline react-components/tests/visual-tests/__screenshots__/Toolbar.stories.spec.ts/cad-model-container-storybook-1.png:1]: Did we need to recreate this screenshot? :thinking: I don't see any difference

**Review by haakonflatval-cognite — APPROVED** [review]
LGTM :+1: Just a couple of comments

File: react-components/tests/visual-tests/__screenshots__/Toolbar.stories.spec.ts/cad-model-container-storybook-1.png
Comment (pramodcog) [reply react-components/tests/visual-tests/__screenshots__/Toolbar.stories.spec.ts/cad-model-container-storybook-1.png:1]: In earlier commits, I had updated `Toolbar` example with `reset camera button`, after `SceneContainer` story was added I moved the example into it, so the change, even though image has not changed, there could be some metadata changes 🤔.

**Pull request merged**